packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/IpNetworkField.tsx

```diff
@@ -73,32 +69,7 @@ function IpNetwork({
             >
                 <section className="ipblock-selector">
                     <div id={id}>
-                        {!prefixMin && (
```

Comment:
See comment above. Are we sure we can lose all of this?

Reply:
Yups